### ISSUE TITLE
Composer deprecation notices

### ISSUE THREAD

**Issue by bradymiller** [openemr/openemr#3498]
Noted the following composer deprecation notices when building openemr:
```
Deprecation Notice: Class OpenEMR\Tests\Fixture\FixtureManagerTest located in ./tests/Tests/Fixtures/FixtureManagerTest.php does not comply with psr-4 autoloading standard. It will not autoload anymore in Composer v2.0. in phar:///usr/bin/composer/src/Composer/Autoload/ClassMapGenerator.php:201
Deprecation Notice: Class OpenEMR\Tests\Services\ProcessingResultTest located in ./tests/Tests/Validators/ProcessingResultTest.php does not comply with psr-4 autoloading standard. It will not autoload anymore in Composer v2.0. in phar:///usr/bin/composer/src/Composer/Autoload/ClassMapGenerator.php:201
Deprecation Notice: Class HTMLPurifier_Language_en_x_test located in ./vendor/ezyang/htmlpurifier/library/HTMLPurifier/Language/classes/en-x-test.php does not comply with psr-0 autoloading standard. It will not autoload anymore in Composer v2.0. in phar:///usr/bin/composer/src/Composer/Autoload/ClassMapGenerator.php:201
```

**Comment by zerai** [comment]
Maybe the namespace is wrong.
First deprecation could be:
```php
namespace Class OpenEMR\Tests\Fixtures\FixtureManagerTest;
```
with final 's' in fixture word.

second deprecation could be:
```php
namespace OpenEMR\Tests\Validators\ProcessingResultTest
```

**Comment by stephenwaite** [comment]
this [PR](https://github.com/openemr/openemr/pull/3500) fixes 1 and 2, thanks @zerai here's #3 status
https://github.com/ezyang/htmlpurifier/issues/246
